facade/interface.go

```diff
@@ -48,24 +48,24 @@ type ValidatorStatisticsProcessor interface {
 // NodeStatusProcessor defines what a node status processor should do
 type NodeStatusProcessor interface {
 	GetNetworkConfigMetrics() (*data.GenericAPIResponse, error)
-	GetNetworkStatusMetrics(shardID uint32) (*data.GenericAPIResponse, error)
+	GetNetworkStatusMetrics(shardID uint32) (*data.GenericAPIResponse, int, error)
 	GetEconomicsDataMetrics() (*data.GenericAPIResponse, error)
```

Comment:
why didn't you add the status code eveywhere?

Reply:
How about even including a `status` field on the API response base?